jobs/ads-incrementality-dap-collector/ads_incrementality_dap_collector/main.py

```diff
@@ -0,0 +1,73 @@
+import click
+import logging
+import traceback
+
+from constants import LOG_FILE_NAME
+from helpers import (
+    get_config,
+    get_experiment,
+    prepare_results_rows,
+    collect_dap_results,
+    write_job_logs_to_bucket,
+    write_results_to_bq,
+)
+
+
+@click.command()
+@click.option("--gcp_project", help="GCP project id", required=True)
+@click.option(
```

Comment:
Additional comment from @gleonard-m: 
Config file and BQ results table will be in different projects.

This `gcp_project` should just be used for the location of the config bucket. Rename to be more specific.

And we need to be able to separately specify the `gcp_project` for the BQ table where the results will be written. That will go in the BQ part of the config json file.